README.md

```diff
@@ -9,7 +9,6 @@ React component styling solved with an elegant ([inspired](#inspiration)) API,
 
 [![Build Status][build-badge]][build]
 [![Code Coverage][coverage-badge]][coverage]
-[![Dependencies][dependencyci-badge]][dependencyci]
```

Comment:
Removing this because I've yet to have a dependency actually fail for me and it's a bit finicky anyway.